### ISSUE TITLE
Alt release triggers switch

### ISSUE THREAD

**Issue by jaredbidlow** [kvakulo/Switcheroo#91]
I just downloaded the latest release, and thanks so much for coming back. I have an issue with the `alt` as `enter` feature in the newest portable release.

I prefer the implementation of having to press enter, but I can probably adapt to a quick switch feature because when I want it, I know that it will bring me to the most recent window. However, I am having trouble using Alt + Tab without accidentally releasing the Alt key before the Tab key. When I release the Alt key, the window switches automatically to the most recent one. I don't see a setting to turn this off, although I have the setting for "Release ..." turned off. 

Further, if I continue to hold down `Alt` after bringing up switcheroo and try to switch windows by pressing 'enter', nothing happens . I tested on the windows native switcher and it is something that they allow. It happens less often than the first issue, but still a small improvement could be made.

Both of these are issues with my use, and maybe others don't experience them.

**Comment by kvakulo** [comment]
Thanks for your feedback, @jaredbidlow! And I'm glad to be back 😃 

I haven't yet been able to reproduce your issue with Alt+Tab, but I'm not surprised if there should be an issue somewhere in the code as it's a bit complicated to catch the right keys, and also the reason that I decided to make the latest version a beta release 😅 

Do I understand correctly that you perform these steps with the `Alt+Tab` feature turned on, but with the auto-switch feature turned off?
1. Press `Alt+Tab`
2. Release `Alt` while `Tab` is still pressed
3. Switcheroo switches to most recent window (this should not happen)

Which version of Windows are your running?

I agree that it should also be possible to switch to a window pressing `Alt+Enter` 👍

**Comment by jaredbidlow** [comment]
That is correct. It switches to the window that the tab-key activity stops on, which could be second recent or farther down the list depending on how long Alt+Tab is held. I am using Windows 7. Ten, I have not tried Switcheroo on.

**Comment by jaredbidlow** [comment]
Yet if press Alt + Tab, then Tab again, the difference between having the auto-switch feature turned on and off becomes apparent. With it on, releasing just Alt will switch to the second window in the list. With it off, it will not switch windows. After the initial Alt + Tab, releasing Alt is switching in both cases, with the setting on or off.

**Comment by csalv5** [comment]
Hi @kvakulo, I'm also experiencing this issue, is there any plan to fix it?
